Makefile

```diff
@@ -133,6 +133,7 @@ image_build:
 	scripts/pull_dockerfile_images.sh
 	sed 's/^FROM .*assisted-service.*:latest/FROM $(subst /,\/,${SERVICE})/' Dockerfile.assisted-test-infra | \
 	 $(CONTAINER_COMMAND) build --network=host -t $(IMAGE_NAME) -f- .
+# 	$(CONTAINER_COMMAND) tag $(IMAGE_NAME) test-infra:latest  # For backwards computability
```

Comment:
backwards compatibility in local envs or something else?

Reply:
I need to uncomment this line, it will tag the `assisted-test-infra:latest`  to `test-infra:latest`.
I just wanted to run the test using the new image name

Reply:
But basically local envs